spring-boot-starter-mesos/src/main/java/com/containersolutions/mesos/config/autoconfigure/TaskActuatorConfiguration.java

```diff
@@ -0,0 +1,41 @@
+package com.containersolutions.mesos.config.autoconfigure;
+
+import com.containersolutions.mesos.scheduler.state.StateRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnClass(HealthIndicator.class)
+@AutoConfigureBefore({org.springframework.boot.actuate.autoconfigure.EndpointAutoConfiguration.class})
+@AutoConfigureAfter({MesosSchedulerConfiguration.class})
+public class TaskActuatorConfiguration {
+    @Autowired
+    StateRepository stateRepository;
+
+    @Value("${mesos.resources.scale:1}")
```

Comment:
`mess.resources.scale` isn't worth much after the application has been started. It's static while the actual scale factor will change. That's why we need a managed bean for the scale factor